plugin-server/src/main/pluginsServer.ts

```diff
@@ -320,58 +314,6 @@ export async function startPluginsServer(
             }
 
             // Below are all legacy consumers that will be replaced by the new ingestion consumer that covers all cases
```

Comment:
nitpick: this comment should also be removed as we're moving to the new ingestion consumer in this PR, right?